__init__.py

```diff
@@ -13,9 +13,12 @@
 import cProfile
 import pstats
 
+from . import addon_updater_ops
+
 # info about add on
 bl_info = {
 	"name": "Fast64",
+	"version": (1, 0, 0),
```

Comment:
so we'll need to update this on each merge?

Reply:
No, if I remember right the updater only uses this as the current version when comparing to tags (like v1.0.1 maybe idk), to determine if there is any new version. But the updater here is configured to not care about versions and only update to the latest main

Reply:
I was somewhat mistaken on this, the answer is still "no" but the

> the updater here is configured to not care about versions and only update to the latest main

part is wrong